Sources/TuistKit/Utils/ManifestGraphLoader.swift

```diff
@@ -94,6 +94,7 @@ final class ManifestGraphLoader: ManifestGraphLoading {
             externalDependencies: dependenciesGraph.externalDependencies
         ) +
             dependenciesGraph.externalProjects.values
+        try GraphLoaderLinter().lintProject(at: path, projects: models)
```

Comment:
Same here, instead of instantiating from the methods, I recommend passing an instance through the constructor so that we can change the instance for unit testing purposes.